addons/website/static/src/js/content/menu.js

```diff
@@ -504,6 +504,37 @@ publicWidget.registry.FadeOutHeader = BaseDisappearingHeader.extend({
     },
 });
 
+publicWidget.registry.menuColor = publicWidget.Widget.extend({
```

Comment:
Seeing this code today, i am not sure I want that much css and js to support this feature... for submenu I'd not recommend to use anyway :grimacing: I'll extract this out of this PR.